src/Components/TicketSelector/TicketSelector.jsx

```diff
@@ -0,0 +1,235 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { CgArrowsExchangeAlt } from "react-icons/cg";
+import { AiOutlineCalendar } from "react-icons/ai";
+import { VscDash } from "react-icons/vsc";
+import { FaUser } from "react-icons/fa";
+import { BiDownArrow } from "react-icons/bi";
+import { theme } from "../../styles/theme";
+import DestinationDesk from "./DestinationDesk";
+
+class TicketSelctor extends Component {
+  constructor() {
+    super();
+    this.state = {
+      depPlace: "",
+      arrPlace: "",
+      // selectDepDate: "",
+      // selectArrDate: "",
+
+      destinationModal: false,
+      arriveModal: false,
+    };
+  }
+
+  openDestination = () => {
+    this.setState({ destinationModal: true });
+  };
+
+  closeDestination = () => {
+    this.setState({ destinationModal: false });
+  };
+
+  openDesk = () => {
+    this.setState({ arriveModal: true });
+  };
+
+  closeDesk = () => {
+    this.setState({ arriveModal: false });
+  };
+
+  selectDepPlace = (e) => {
+    this.setState({ depPlace: e.target.innerHTML });
+  };
+
+  selectArrPlace = (e) => {
+    this.setState({ arrPlace: e.target.innerHTML });
+  };
+
+  handleChangeInput = () => {
+    if (this.state.depPlace) {
+      this.setState({ depPlace: this.state.arrPlace });
+    }
+  };
+
+  render() {
+    console.log("dep>>>", this.state.depPlace);
+    console.log("arr>>>", this.state.arrPlace);
+    const {
+      depPlace,
+      arrPlace,
+      selectDepDate,
+      selectArrDate,
+      destinationDesk,
+      destinationModal,
+      arriveModal,
+    } = this.state;
+    return (
+      <>
+        {destinationModal && (
+          <DestinationDesk
+            top={"50px"}
+            left={"10px"}
+            closeDestination={this.closeDestination}
+            selectDepPlace={this.selectDepPlace}
+          />
+        )}
+
+        {arriveModal && (
+          <DestinationDesk
+            top={"50px"}
+            left={"220px"}
```

Comment:
```suggestion
          <DestinationDesk
            top="50px"
            left="220px"
```
자바스크립트 문법을 사용해야할 때만 중괄호를 열고 값을 입력하시면 됩니다. 일반 텍스트의 경우 직접 쌍따옴표를 사용해 전달하면 가능합니다.

Reply:
@14-yoonl 이거 접혀있어서 못보셨을거 같은데 혹시 확인하셨을까요?